### ISSUE TITLE
bug: dependency built with stencil, used in react project, the event triggered twice!

### ISSUE THREAD

**Issue by fayeah** [ionic-team/stencil#3511]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/stencil/blob/main/.github/CONTRIBUTING.md).
- [X] I agree to follow the [Code of Conduct](https://github.com/ionic-team/stencil/blob/main/CODE_OF_CONDUCT.md).
- [X] I have searched for [existing issues](https://github.com/ionic-team/stencil/issues) that already report this problem, without success.

### Stencil Version

2.13.0

### Current Behavior

the details please check 'Steps to Reproduce' part.

### Expected Behavior

click event should be triggered only once.

### Steps to Reproduce

step1: create a component with stencil, let's call it `TestClick`;
step2: add event for `TestClick`;
```
// ok event
 @Event({
    eventName: 'ok',
    composed: false,
    cancelable: false,
    bubbles: false,
  })
  onOk: EventEmitter<number>;

private _onOk = async (e) => {
    e.preventDefault();
    const okEv = this.onOk.emit('some value');
    if (!okEv.defaultPrevented) {
      console.log('not defaultPrevented');
      return;
    }
  };

```
step3: use this component (`TestClick`) in **another** react project;
step4: add event click function, let's call it `handleClick`;
```
// use TestClick
 <TestClick onOk={(e) => handleClick(e)}></TestClick>

// handler
 const handleClick = (e) => {
 console.log('aaaaaa');
  };
```

then the log message 'aaaaaa' was logged twice, which means handleClick was called twice, not sure whether it is bubble issue or not, but this is a bit weird.

### Code Reproduction URL

not now

### Additional Information

please help to check, thanks.

**Comment by rwaskiewicz** [comment]
I'm going to close this issue due to lack of reproduction case. If this is still an issue, please feel free to create a new issue with a minimal reproduction URL for the team to pull down and take a look at. Thanks!
